client/container.go

```diff
@@ -656,6 +656,11 @@ func (c *Client) ContainerSetEACL(ctx context.Context, prm PrmContainerSetEACL)
 		panic("eACL table not set")
 	}
 
+	_, isCIDSet := prm.table.CID()
+	if !isCIDSet {
+		return nil, errors.New("eaclTable doesn't contain container identifier")
```

Comment:
```suggestion
		return nil, errors.New("missing container in eACL table")
```

btw currently we panic on invalid input (see above).

this requirement should be mentioned in https://pkg.go.dev/github.com/nspcc-dev/neofs-sdk-go/client#PrmContainerSetEACL.SetTable.